mungegithub/mungers/test-fixtures/path-label.txt

```diff
@@ -0,0 +1,25 @@
+# This file is used by the path-label munger and is of the form:
```

Comment:
Instead of creating a separate sample file for testing please move the file contents to a `const string` defined in `path_label_test.go` so that the sample that the test is dependent on is hard coded with the test.